sklearn/preprocessing/data.py

```diff
@@ -522,30 +634,62 @@ class MaxAbsScaler(BaseEstimator, TransformerMixin):
     ----------
     scale_ : ndarray, shape (n_features,)
         Per feature relative scaling of the data.
+
+    max_abs_ : ndarray, shape (n_features,)
+        Per feature maximum absolute value.
+
+    n_samples_seen_ : int
+        The number of samples processed by the estimator. Will be reset on
+        new calls to fit, but increments across ``partial_fit`` calls.
     """
 
     def __init__(self, copy=True):
         self.copy = copy
 
     def fit(self, X, y=None):
-        """Compute the minimum and maximum to be used for later scaling.
+        """Compute the maximum absolute value to be used for later scaling.
 
         Parameters
         ----------
-        X : array-like, shape [n_samples, n_features]
+        X : array-like or CSR or CSC matrix with shape [n_samples, n_features]
             The data used to compute the per-feature minimum and maximum
             used for later scaling along the features axis.
         """
+        return self.partial_fit(X, y)
+
+    def partial_fit(self, X, y=None):
+        """Online computation of the max absolute value on X for later scaling.
+        All of X is processed as a single batch. This is intended for cases
+        when `fit` is not feasible due dimensionality of X and/or because X is
+        read from a continuous stream.
+
+        Parameters
+        ----------
+        X : array-like or CSR or CSC matrix with shape [n_samples_, n_features]
```

Comment:
Can you change this to the same style as others?

X : {array-like, sparse matrix}